web/templates/gear_menu.hbs

```diff
@@ -176,7 +176,7 @@
                 </a>
             </li>
             <li role="presentation">
-                <a href="#logout" class="logout_button hidden-for-spectators" role="menuitem">
+                <a class="logout_button hidden-for-spectators" role="menuitem">
```

Comment:
This probably should have been in its own PR, or a prep commit at the beginning, since it doesn't depend on the rest, but in any case, rebased to the start and merged as 55b0b5cb965b7dee850244baf55a4d96e5e465c1.